packages/babel-helper-create-class-features-plugin/src/index.js

```diff
@@ -30,6 +30,8 @@ export { FEATURES, injectInitialization };
 const version = pkg.version.split(".").reduce((v, x) => v * 1e5 + +x, 0);
 const versionKey = "@babel/plugin-class-features/version";
 
+const privateNamesStack = [new Map()];
```

Comment:
When using a `WeakSet`, please put this in the `createClassFeaturePluign` function so that when the plugin finishes running it can be garbage-collected, even if the AST was still retained.